packages/web-components/src/components/ic-horizontal-scroll/ic-horizontal-scroll.tsx

```diff
@@ -42,6 +42,11 @@ export class HorizontalScroll {
    */
   @Prop() appearance?: IcThemeForeground = "default";
 
+  /**
+   * @internal The event to listen to that will trigger focus behavior.
```

Comment:
This comment is phrased a bit strangely

Reply:
"the event to listen to in order to trigger focus handler logic" ?

Reply:
Sorry, i should have provided a suggestion myself! "the name (as a string) of the event that triggers focus handler logic" ?